src/App.jsx

```diff
@@ -14,70 +15,71 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export default function App() {
+  const jam = goods.find(e => e === 'Jam');
+  const [item, setItem] = useState(jam);
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  function itemPick(elem) {
+    setItem(elem);
+  }
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  function itemClear() {
+    setItem('');
+  }
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+  return (
+    <main className="section container">
+      {item === '' ? (
+        <h1 className="title is-flex is-align-items-center">
+          No goods selected
+        </h1>
+      ) : (
+        <h1 className="title is-flex is-align-items-center">
+          {`${item} is selected`}
+          <button
+            onClick={() => itemClear()}
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+          />
+        </h1>
+      )}
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+      <table className="table">
+        {goods.map(elem => (
+          <tr
+            key={elem}
+            data-cy="Good"
+            className={item === elem ? 'has-background-success-light' : ''}
+          >
+            <td>
+              {item === elem ? (
```

Comment:
you use this check `item === elem` two times

create a const `isItemSelected` for this inside map